### PR TITLE
Added support Maintenance

### PR DESCRIPTION
**Pull request by LazarenkoA**

### CODE REVIEW

File: maintenance.go [hunk 1]
```diff
@@ -125,15 +126,18 @@ func (m *MaintenanceCreateParams) FillHostIDs(session *Session) error {
 		return err
 	}
 
+	err = errors.New("Failed to tind ID by host name")
```
Comment (cavaliercoder) [inline maintenance.go]: Typo: "tind".

Please also make this error a global var, maybe `ErrMaintenanceHostNotFound` or something to that effect.

Comment (LazarenkoA) [reply maintenance.go]: done

File: maintenance.go [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+)
+
+type MaintenanceType int
+type TagsEvaltype int
+
+var ErrNotHostFound = errors.New("Failed to tind ID by host name")
```
Comment (cavaliercoder) [inline maintenance.go]: One last nit: Please replace "tind" with "find".

**Comment by cavaliercoder** [comment]
Nice work, thank you!

**Pull request merged**